src/doc/style-guide/src/items.md

```diff
@@ -505,10 +505,6 @@ use b;
 Because of `macro_use`, attributes must also start a new group and prevent
 re-ordering.
 
-Note that tools which only have access to syntax (such as `rustfmt`) cannot
```

Comment:
Isn't this true? `rustfmt` doesn't have any way of telling apart some import like `use c::*;` whether it comes from `mod c;` or `extern crate c;`.

Reply:
It's partly true, insofar as rustfmt doesn't actually do name resolution, but I think it's misleading, insofar as rustfmt can still do useful things like "is it literally `std` or `alloc` or `core`, or `crate`". And I don't think it's helping make a useful point in the style guide here, nor should we be trying to document the limitations of any particular tool here.